module/data/chat-message/request-message-data.mjs

```diff
@@ -87,7 +90,7 @@ export default class RequestMessageData extends ChatMessageDataModel {
    */
   static async #handleRequest(event, target) {
     const actor = fromUuidSync(target.closest("[data-uuid]").dataset.uuid);
-    const result = await CONFIG.DND5E.requests[this.handler](actor, this.parent, this.data);
+    const result = await CONFIG.DND5E.requests[this.handler](actor, this.parent, this.data, { event });
```

Comment:
Right now the type for `CONFIG.DND5E.requests` is just `Record<string, Function>`, it would be good to properly document the expected API for the callback.